networks/osmosis-1/upgrades/v12/v12_binaries.json

```diff
@@ -1,6 +1,6 @@
 {
     "binaries": {
-        "linux/amd64": "",
-        "linux/arm64": ""
+        "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v12.0.0/osmosisd-12.0.0-linux-amd64?checksum=sha256:d01423cf847b7f95a94ade8811bbf6dd9ec5938d46af0a14bc62caaaa7b7143e",
```

Comment:
```
roman@akhtariev /mnt/c/Users/roman/osmo-v12 $ sha256sum osmosisd-12.0.0-linux-amd64
34017d81541f7cbfd64076cf07c65f71174266367f7007be42bb78f2a7a28d73  osmosisd-12.0.0-linux-amd64
roman@akhtariev /mnt/c/Users/roman/osmo-v12 $ sha256sum osmosisd-12.0.0-linux-arm64
34017d81541f7cbfd64076cf07c65f71174266367f7007be42bb78f2a7a28d73  osmosisd-12.0.0-linux-arm64
```

Debugging further

Reply:
OK. This seems to be the problem:

https://github.com/osmosis-labs/osmosis/pull/2842

We are building amd64 as arm64. That's why the binaries and their hashes are the same for both platforms. I'm not sure why your hash here is different.

I attached the manually rebuilt binaries to the release draft. The old ones (auto-generated) are still there with an "old" suffix just in case but we should remove them.

I have tried regenerating the binaries twice and got the following hashes both times:
```
a4cf38ff80f50f3b7203d929192c9a3c822fb3a091d42342a36f3e236446ed14  osmosisd-12.0.0-linux-amd64
75e724222b3c767d021539b166157d15bd97d72ed5962785add797e981ccc7a5  osmosisd-12.0.0-linux-amd64.tar.gz
fe156971edcd6bd8c1fe462360e1d1f5cd4720e1c66ec5d03cd7de03eeb8cefd  osmosisd-12.0.0-linux-arm64
d9b6605ce996545f23f927d961c43d0486faa38b9930af48e516ad1705e62bb2  osmosisd-12.0.0-linux-arm64.tar.gz
```

Reply:
```
8bbac5ab09b45403ef8270f9d5077571012b4024723e138b50dc31beb3c96d4e  osmosisd-linux-amd64
561786230c0c9db5bcace506bf5b8e1018156ed802832cdfe1ba694a68155645  osmosisd-linux-amd64.tar.gz
34017d81541f7cbfd64076cf07c65f71174266367f7007be42bb78f2a7a28d73  osmosisd-linux-arm64
97b0972f9acdf09d885abbff6cf7f86a4ecd3e5895d99de143fef6a99004cdcf  osmosisd-linux-arm64.tar.gz
```